Fixing the front snapshot index not adding

cfumiwu opened the pull request:
Fixing the graph panel won't scroll correctly once it reaches the end snapshot and try to step back and replay.

cfumiwu committed:
Fixing the front snapshot index not adding

earthling-amzn reviewed (commented):
Can you explain for me what the problem was and how this fixes it?

cfumiwu commented:
So as I used before the problem occurs when it reaches the very end and I tried to scroll back, and the graph panel looks like it is stuck in the same frame and not scrolling properly. Therefore, when I was investigating the problem yesterday. I tried to print out the `frontSnapshotIndex` and `endSnapshotIndex` and I figure out that when the visualizer scroll back it will go into the first part of the if statement for the `run()` method. However, because it is not adding the `frontSnapshotIndex` here, the graph panel couldn't scroll properly. Therefore, I added this statement to add the `frontSnapshotIndex` and fixed the problem.
